cm/README.md

```diff
@@ -0,0 +1,11 @@
+# Git Sync for Demo Portal deployments
+
+## When deployed to AKS from the Demo Portal, files from this folder will be copied to the website root folder automatically
```

Comment:
Is this JUST when deploying from the portal? Doesn't the CM always look at the repo, even if it's deployed using docker-compose? I think we can remove the "When deployed to AKS from the Demo Portal"?

Reply:
Good point, updated.